docs/_includes/generated/rest_v1_dispatcher.html

```diff
@@ -1051,7 +1051,7 @@
     <tr>
       <td colspan="2">
         <ul>
-<li><code>mode</code> (optional): String value that specifies the termination mode. Supported values are: "cancel, stop".</li>
+<li><code>mode</code> (optional): String value that specifies the termination mode. Supported value is: "cancel".</li>
```

Comment:
As the html file located in `docks/_includes/generated`, maybe you should change the source file `TerminationModeQueryParameter` and regenerate the html file instead of changing the html file directly.

Reply:
> Also as the java doc said this is for the legacy usage, do we need to change it?

I think the legacy usage is for `cancel` mode now. Since `stop` mode throws an exception, why we should document it for new users?

Reply:
@klion26 I do not find the same description what I changed in `TerminationModeQueryParameter`, can you tell me what I should change in `TerminationModeQueryParameter ` and how to regenerate the HTML file? Thanks.